### DISCUSSION TITLE
How do I hide posts from a particular person?

### DISCUSSION THREAD

**Discussion by digeomel** [dailydotdev/daily#1309] [Feature Requests]
It's not a group, it's a person that I want to hide posts from.

**Comment by rajneesh-18** [comment]
> It's not a group, it's a person that I want to hide posts from.

So, what do you think. How can it be solved?

**Comment by idoshamun** [comment]
Closing as it's a dup of https://github.com/dailydotdev/daily/discussions/514.
Unfortunately it's still not in development
